[IMP-FIX] product_variant_custom_sale: selection of attributes to show

mikelarre opened the pull request:
in description and fixed description minor issue

mikelarre committed:
[IMP] version modules security changes

mikelarre committed:
[IMP] product_variant_custom_sale: allowed product attribute line
editing in 'sent' state sale lines

mikelarre committed:
[IMP] product_variant_custom_sale: sale_line_description

mikelarre committed:
[FIX] product_variant_custom_sale: create button in sale line

mikelarre committed:
Merge branch '12.0' of https://github.com/avanzosc/mrp-addons into 12.0

mikelarre committed:
[FIX-IMP] product_version modules: posibility to add product version in new picking + move menu of purchase version + product version onchange fix + AttributeError fix

mikelarre committed:
[FIX+IMP] many version modules fixes: permissions + sale line descriptions + minor changes. Added new field in pickings to assign product version when picking is created manuali.

mikelarre committed:
[FIX] mrp_production_custom_variants_inherite_attributes: security
changes

mikelarre committed:
[FIX] version modules security file issues

mikelarre committed:
Merge branch '12.0' of https://github.com/avanzosc/mrp-addons into 12.0

mikelarre committed:
[FIX] product_variant_custom_stock: model security file fixed

mikelarre committed:
Merge branch '12.0' of https://github.com/avanzosc/mrp-addons into 12.0

mikelarre committed:
Merge branch '12.0' of https://github.com/avanzosc/mrp-addons into 12.0

mikelarre committed:
[FIX] version_modules: function relocate, security file fix, sale
description

mikelarre committed:
Merge branch '12.0' of https://github.com/avanzosc/mrp-addons into 12.0

mikelarre committed:
Merge branch '12.0' of https://github.com/avanzosc/mrp-addons into 12.0

mikelarre committed:
[IMP] version modules: add boolean in value lines to show in line
description

mikelarre committed:
Merge branch '12.0' of https://github.com/avanzosc/mrp-addons into 12.0

mikelarre committed:
Merge branch '12.0' of https://github.com/avanzosc/mrp-addons into 12.0

mikelarre committed:
[FIX] product_variant_custom_sale: product sale line description

alfredoavanzosc reviewed (approved):
:+1:

oihane commented on product_variant_custom_sale/models/sale.py:
```diff
@@ -73,14 +73,16 @@ class SaleOrderLine(models.Model):
 
     def _get_sale_line_description(self):
-        product_lang = self.product_id.with_context(
+        product = self.product_id or self.product_tmpl_id
```
Y no convendría tener en cuenta el método: 

[`def get_sale_order_line_multiline_description_sale(self, product):`](https://github.com/OCA/OCB/blob/12.0/addons/sale/models/sale.py#L1696)

mikelarre replied:
Ok esta revisado y se han añadido nuevos cambios. Botón para ver los quants de los productos planificados, copiar los valores de los atributos y el template al mo cuando se confirma un pedido y otros cambios menores

mikelarre committed:
Merge branch '12.0' of https://github.com/avanzosc/mrp-addons into 12.0

mikelarre committed:
[FIX] version modules: description changes in sale lines, non editable
attribute values issue in purchase lines, on mto create correctly mo product
attribute values
